enzy_htp/structure/metal_atom.py

```diff
@@ -125,34 +143,36 @@ def is_metal_center(self):
         (place holder for override)
         TODO dispatch and make more consistant way to determine for 'boundary' metals like Mg2+"""
         return self.name in chem.METAL_CENTER_MAP
+
     #endregion
-    
+
     # TODO below to change
 
-    def location(self) -> np.array: #@shaoqz: maybe unify as coord etc. TODO
+    def location(self) -> np.array:  #@shaoqz: maybe unify as coord etc. TODO
         """Gets the location of the MetalUnit."""
         return self._atoms[0]
 
-
     def get_radii(self, method: str = "ionic") -> Union[float, None]:
         """Gets the atomic radii for the metal using specified method.
         Allowed values are "ionic" or "vdw" for ionic and van-der waals, respectively.
         Returns a float or None if the metal does not have a distance.
         """
-        return chem.get_atom_radii(self.atom_name_, method)
+        tks = list(self._name.lower())
+        tks[0] = tks[0].upper()
+        return chem.get_atom_radii(''.join(tks), method)
```

Comment:
I don't think the original method have any problem. But the new solution is also ok

Reply:
It was having problems in the tests

Reply:
Hmmmm that might suggest there are errors from other part. Which test is that?

Reply:
let me check this out

Reply:
Oh its that we need to use element instead of atom name. fixed it